spyder/config/base.py

```diff
@@ -229,6 +229,7 @@ def get_image_path(name, default="not_found.png"):
 LANGUAGE_CODES = {'en': u'English',
                   'fr': u'Français',
                   'es': u'Español',
+                  'hu': u'Hungarian',

```

Comment:
Thanks a lot for this :-)

Now, the idea here is to use the Native language name, I am guessing `Magyar` ?